person.rb

```diff
@@ -1,6 +1,8 @@
 require_relative 'nameable'
 
 class Person < Nameable
+  has_many :rentals
+
```

Comment:
- [ ] Great job working on the association so far but I think it would be great if you initialized the rentals here so kindly add it here in your constructor 🔥✅💯 here's a link for a [lesson](Link to the example about constructors in a previous [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) to understand better)

![image](https://github.com/jkanyi-web/OOP_School_Library/assets/58553711/7119ca34-b241-4f7a-8283-f67c18e0c090)